LibreNMS/Alert/RunAlerts.php

```diff
@@ -131,6 +132,16 @@ public function describeAlert($alert)
         }
         $extra = $alert['details'];
 
+        // finds all applications for the device and includes those
```

Comment:
This could be replaced with:
```php
$obj['applications'] = $device->applications->groupBy('app_type')
```

keep in mind you can have more than one application for a device of a single type.  That is why I used groupBy instead of keyBy

Reply:
Note to self, we need to change these so they are only fetched when used.

Reply:
> This could be replaced with:
> 
> ```
> $obj['applications'] = $device->applications->groupBy('app_type')
> ```
> 
> keep in mind you can have more than one application for a device of a single type. That is why I used groupBy instead of keyBy

Does anything actually use this?

Was actually looking at seeing if they column in question would be removed.

Reply:
It looks like drbd and proxmox use it sadly.

Those should be re-written to use app data at some point.

Reply:
proxmox does as well

Honestly both of those could really use a massive re-write looking at the code in question.

Reply:
@murrant Okay, looking into it that requested change does not break anything. Just adds in a annoying [0] to be used till those apps are re-written. That said neither proxmox or drdb actually produce any meaningfully usable data.

Reply:
And done. Docs updated.